shared/editor/embeds/GoogleLookerStudio.tsx

```diff
@@ -3,30 +3,30 @@ import Frame from "../components/Frame";
 import Image from "../components/Image";
 import { EmbedProps as Props } from ".";
 
-function GoogleDataStudio(props: Props) {
+function GoogleLookerStudio(props: Props) {
   return (
     <Frame
       {...props}
       src={props.attrs.href.replace("u/0", "embed").replace("/edit", "")}
       icon={
         <Image
-          src="/images/google-datastudio.png"
-          alt="Google Data Studio Icon"
+          src="/images/google-lookerstudio.png"
+          alt="Google Looker Studio Icon"
           width={16}
           height={16}
         />
       }
       canonicalUrl={props.attrs.href}
-      title="Google Data Studio"
+      title="Google Looker Studio"
       border
     />
   );
 }
 
-GoogleDataStudio.ENABLED = [
+GoogleLookerStudio.ENABLED = [
   new RegExp(
-    "^https?://datastudio\\.google\\.com/(embed|u/0)/reporting/(.*)/page/(.*)(/edit)?$"
+    "^https?://lookerstudio\\.google\\.com/(embed|u/0)/reporting/(.*)/page/(.*)(/edit)?$"
```

Comment:
Can we retain support for the old embeds?

Reply:
I have tried something below but am not sure about how they are kept in the database.
Maybe it's more complex than a simple regex OR... And requires a kind of migration ?

Reply:
This is fine 👍

Reply:
Great !